### ISSUE TITLE
Nan in summary histogram error for training images if faster_rcnn_resnet101_coco_11_06_2017 model is used.

### ISSUE THREAD

**Issue by rashikcs** [tensorflow/tensorboard#744]
------------------------

### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**:
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Ubuntu 16.04
- **TensorFlow installed from (source or binary)**: binary
- **TensorFlow version (use command below)**: 1.3.0
- **Python version**: 3.7
- **Bazel version (if compiling from source)**:
- **GCC/Compiler version (if compiling from source)**:
- **CUDA/cuDNN version**: cudnn-8.0
- **GPU model and memory**:  GeForce GTX 1060 6GB 
- **Exact command to reproduce**: python train.py --logtostderr --train_dir=training/ (path to training directory) --pipeline_config_path=training/faster_rcnn_resnet101.config (path to .config file)

### Describe the problem

InvalidArgumentError (see above for traceback): Nan in summary histogram for: FirstStageFeatureExtractor/resnet_v1_101/block3/unit_6/bottleneck_v1/conv2/weights_1
	 [[Node: FirstStageFeatureExtractor/resnet_v1_101/block3/unit_6/bottleneck_v1/conv2/weights_1 = HistogramSummary[T=DT_FLOAT, _device="/job:localhost/replica:0/task:0/cpu:0"](FirstStageFeatureExtractor/resnet_v1_101/block3/unit_6/bottleneck_v1/conv2/weights_1/tag, FirstStageFeatureExtractor/resnet_v1_101/block3/unit_6/bottleneck_v1/conv2/weights/read)]]
	 [[Node: FirstStageFeatureExtractor/resnet_v1_101/block3/unit_16/bottleneck_v1/conv3/BatchNorm/gamma/read/_777 = _Recv client_terminated=false, recv_device="/job:localhost/replica:0/task:0/gpu:0", send_device="/job:localhost/replica:0/task:0/cpu:0", send_device_incarnation=1, tensor_name="edge_2768_FirstStageFeatureExtractor/resnet_v1_101/block3/unit_16/bottleneck_v1/conv3/BatchNorm/gamma/read", tensor_type=DT_FLOAT, _device="/job:localhost/replica:0/task:0/gpu:0" ()]]

**Comment by jart** [comment]
Could you include the full error including traceback and stuff?

**Comment by rashikcs** [comment]
Here are the full log file of the specific command and error  including traceback and stuff.

[Error.txt](https://github.com/tensorflow/tensorboard/files/1477403/Error.txt)

**Comment by jart** [comment]
One thing to note is that NaNs are generally a sign that there's a problem in your model. Can you make the model not produce NaNs? If not, then what do you want to see TensorBoard do with NaN data in histograms and why do you feel it would be useful?

**Comment by rashikcs** [comment]
The same dataset is being used to train model using ssd_inception_v2 &  ssd_mobilenet_v1 file. Then it trained properly but whenever i am using other than those two it is producing this error.

**Comment by jart** [comment]
The best place to get support is [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since this is not a bug or feature request.

**Comment by taewookim** [comment]
I have the same problem. Using a dataset that trained perfectly fine on SSD mobilenet, but produces this exact same error on Faster RCNN resnet 101

**Comment by taewookim** [comment]
1) this is not a tensorboard issue
2)  this happens on Faster RCNN resnet 50 as well as any other faster_rcnn_XXX models from the detection model zoo

**Comment by auroua** [comment]
I encountered the same issue. I run the code in `python3.5+tf1.4.1` and have no problem, but when I run the same code in `python3.6+tf1.3` this error printed. I think this is a bug.

**Comment by nfelt** [comment]
If you can establish that it's a TensorFlow bug then it's worth opening an issue, but that would be in the TensorFlow issue tracker: https://github.com/tensorflow/tensorflow/issues  The TensorBoard failure is just a symptom of the numerical instability in the underlying TensorFlow model code.
